yarn.lock

```diff
@@ -10758,6 +10758,11 @@ jest-snapshot@^27.0.6:
     pretty-format "^27.0.6"
     semver "^7.3.2"
 
+jest-svg-transformer@^1.0.0:
```

Comment:
Was importing this package removed? If so I think you can revert this change as well